docs/eloquent-models/model-class.txt

```diff
@@ -249,8 +249,8 @@ The following code example shows mass assignment of the ``Planet`` model:
 .. code-block:: php
 
    $planets = [
-       [ 'name' => 'Earth', gravity => 9.8, day_length => '24 hours' ],
-       [ 'name' => 'Mars', gravity => 3.7, day_length => '25 hours' ],
+       [ 'name' => 'Earth', 'gravity' => 9.8, 'day_length' => '24 hours' ],
+       [ 'name' => 'Mars', 'gravity' => 3.7, 'day_length' => '25 hours' ],
```

Comment:
I: based on the model definition and context, i think this is the correct field name
S:

```suggestion
       [ 'name' => 'Earth', 'gravitational_force' => 9.8, 'day_length' => '24 hours' ],
       [ 'name' => 'Mars', 'gravitational_force' => 3.7, 'day_length' => '25 hours' ],
```